### DISCUSSION TITLE
Disable fractions view

### DISCUSSION THREAD

**Discussion by theorician** [mealie-recipes/mealie#3992] [General]
Is it possible to disable showing ingredients as fractions? Fractions are only really useful for US/Imperial measurements, but for people that use mass scales, being told to add 2 5/7th grams of cumin to a recipe is counterproductive... 

I see that the json view has a fraction: true/false field, but I don't see how I can disable either this globally - or even in the UI without editing the json for every field.

Thank you :)

**Comment by qb42** [comment]
You can edit the settings for individual units at /group/data/units

**Reply by theorician** [reply]
Thank you, that did it! 

(Feedback: would be great if I could mark multiple and turn on/off 'use fractions' and 'use abbreviations' from the drop down, rather than just multi-delete.)

**Comment by Ronaldvr** [comment]
I see something similar that seems to be missing: Some foodstuffs or units (pinch) are indivisible (you can only have a whole or it is wholly impractical to try and cut something e.g a peppercorn). So to accommodate that there would need to be a setting with both units and foodstuff that allows for this. Now If I dial down a recipe for 4 to 1 person I get for example 1/4 pinch, 1/4 peppercorn which is a bit strange
